src/main/java/nl/novi/techiteasydto/controllers/TelevisionController.java

```diff
@@ -0,0 +1,37 @@
+package nl.novi.techiteasydto.controllers;
+
+import nl.novi.techiteasydto.services.TelevisionService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(name = "televisions")
```

Comment:
Probeer consistent een / te gebruiken in je mappings, ook je algemene RequestMapping